samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorMessages_eu.properties

```diff
@@ -876,3 +876,6 @@ fixed_and_random_draw_msg_no_date=Parte honetako galderak bi modutan jaso dira:
 edit_published_assessment_warn_edit_pool_fixed_questions=Aldaketak eragina izango dute azterketa honetan soilik, ez erabilitako biltzaileetan. "{0}" izeneko biltzailean dauden galdera finkoak editatzeko sakatu goiko esteka, Galderak, orrialde honetan goian dagoena. Gauza bera egin editatzeko "{1}" izeneko biltzailean dauden galderak.
 notfixedquestionselected_error=Gutxienez galdera finko bat hautatu behar duzu.
 notenough_error=Ez dago galdera nahikorik biltzailean. Galdera finkoen eta ausaz ateratako galderen baturak ez du gainditu behar erabilitako biltzailean daudenak.
+
+# S2U-20
+random_multiple_pools=Random draw from multiple question pools
```

Comment:
@pablosanroman could you please translate this?

Reply:
Hi @bgarciaentornos !!
random_multiple_pools=Ausazko bista biltzaileetatik